model/autosklearn/setup.py

```diff
@@ -0,0 +1,16 @@
+import os
+import importlib.util
+from setuptools import setup
+
+# Boilerplate to load commonalities
+spec = importlib.util.spec_from_file_location(
+    "setup_common", os.path.join(os.path.dirname(__file__), "setup_common.py")
+)
+common = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(common)
+
+common.KWARGS["entry_points"] = {
+    "dffml.model": [f"misc = {common.IMPORT_NAME}.misc:MiscModel"]
```

Comment:
This will of course need updating

Reply:
yes. for now i have removed it. i will add to it once some models are implemented